pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -64,11 +64,15 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 		return err
 	}
 
-	// As mostly all LoadBalancer features are only supported on "amphora" we explicitly set the provider
-	// in the LoadBalancer create call to make sure to get the desired features - even if multiple providers exist.
+	// Choose the selected provider if it is set in cluster spec, if not we explicitly set the provider to "amphora"
```

Comment:
See above comment about this. Should this be 'Amphora' or can we reasonably not specify anything and let Octavia pick? I don't know enough (at least not yet) to pick, but I would like to know that it we have considered the options and made a deliberate choice.

Reply:
I believe there is a `default_provider_driver` setting in Octavia defaults to `amphora`.
https://docs.openstack.org/octavia/latest/configuration/configref.html

IMO, it's reasonable to leave it empty and let Octavia use its default provider

Reply:
If Octavia has its own default I prefer to use that rather than encode our own.